tests/nucleo-g474_w_jumpers.rs

```diff
@@ -0,0 +1,316 @@
+#![no_std]
```

Comment:
@AdinAck 
> This looks great! Have you run any integration tests?

I have now added some more tests. These ones require a jumper(I took the one at CN12 at the top left corner) when using a Nucleo-G474 between A1 and A2. But I do believe that single modification opens up for some pretty interesting self tests